config/locales/nl.yml

```diff
@@ -0,0 +1,16 @@
+nl:
+    revise_auth:
+        account_deleted: "Uw account is verwijderd."
+        account_updated: "Account succesvol bijgewerkt."
+
+        invalid_email_of_password: "Ongeldige e-mail of wachtwoord."
+        sign_up_or_login: "Aanmelden of inloggen om door te gaan."
+
+        # Password changes
+        password_changed: "Uw wachtwoord is succesvol gewijzigd."
+        incorrect_password: "Uw huidige wachtwoord is onjuist. Probeer het opnieuw."
+
+        # E-mail confirmations
+        email_confirmed: "Uw e-mailadres is succesvol bevestigd."
+        email_confirm_failed: "E-mail adres bevestigen niet mogelijk."
```

Comment:
with and without space ? Is it legit in nl ? `e-mailadres` and `E-mail adres`

Reply:
Should indeed be without a space! Probably auto correct, fixed it. Thanks for notifying.